packages/react-components/src/components/styles/ReactionOverlay.style.ts

```diff
@@ -184,14 +184,18 @@ export interface IReactionStyleBucket {
  * height, width, and opacity.
  * @private
  */
-export function getReactionStyleBucket(activeSpritesCount: number): IReactionStyleBucket {
-  const index = activeSpritesCount < 3 ? 0 : activeSpritesCount < 5 ? getRandomInt(0, 1) : getRandomInt(0, 2);
+export function getReactionStyleBucket(): IReactionStyleBucket {
+  //const index = activeSpritesCount < 3 ? 0 : activeSpritesCount < 5 ? getRandomInt(0, 1) : getRandomInt(0, 2);
+  // Having dynamic emoji size on rendering animation impacts performance of the animation itself.
+  // So we are choosing to use a fixed size for all cases.
+  const index = 1;
   return ReactionStyleBuckets[index];
```

Comment:
Is this always `[1]` now?
If so remove this function and just make a const 
```
const reactionPresentationStyles = {
   sizeScale: 0.9,
    heightMaxScale: 0.7 * 0.95,
    opacityMax: 0.9
}
```

Reply:
I would like to keep them in the array. So If we need to update the styles or values, we can add or remove them without editing the component files. Makes it easier to test and find bugs.

Reply:
cleaned up other styles. 
Documented the reasons for style value being in the bucket array.